dhis2/src/main/java/org/motechproject/dhis2/service/impl/SettingsServiceImpl.java

```diff
@@ -60,6 +61,14 @@ private synchronized void loadSettings() {
             LOGGER.debug("Loading DHIS2 settings...");
             Gson gson = new Gson();
             settings = gson.fromJson(jsonText, Settings.class);
+            if (settings.getServerURI() != null) {
+                URI uri = new URI(settings.getServerURI());
+                uri = uri.normalize();
+                String uriString = uri.toString();
+                if (uriString.endsWith("/")) {
```

Comment:
please move this to separate method 'removeTrailingSlash'. The code will be more readable